configure.ac

```diff
@@ -72,7 +72,6 @@ CFLAGS_COMMON+=" -Wimport"
 CFLAGS_COMMON+=" -Winit-self"
 CFLAGS_COMMON+=" -Winline"
 CFLAGS_COMMON+=" -Winvalid-pch"
-CFLAGS_COMMON+=" -Wlong-long"
```

Comment:
why removed this ?

Reply:
ULL (unsigned long long) is valid in 32 bit arch. -Wlong-long will throw error if ULL is used.

Reply:
for 64bit value ULL is required in 32bit arch

Reply:
Place it is needed in SAI.
https://github.com/opencomputeproject/SAI/blob/55df00a5b4e7a89c2576e2ad3174242d948261ec/meta/saiserialize.c#L853

Reply:
its valid on 32bit arch, but not valid according to ansi standard

Reply:
Need this to support 32bit arch compilation